python/ray/experimental/workflow/workflow_storage.py

```diff
@@ -460,14 +467,68 @@ def get_latest_progress(self) -> "StepID":
         return asyncio_run(self._get(self._key_workflow_progress(),
                                      True))["step_id"]
 
+    def _put_object_ref(self, obj_ref: ObjectRef,
+                        upload_tasks: List[ObjectRef]) -> str:
+        @ray.remote
+        def put_helper(loc: str, obj: Any, wf_storage: WorkflowStorage) -> str:
+            asyncio.get_event_loop().run_until_complete(
+                wf_storage._put(loc, obj))
```

Comment:
Question here: loc is a str, but key for wf_store._put should be a list. Anything wrong here?

Reply:
Yeah this surprisingly worked (made each character in the string its own directory) 😅. Fixed